tests/Fixtures/TestBundle/Entity/Foo.php

```diff
@@ -42,15 +41,13 @@ class Foo
      * @var string The foo name
      *
      * @ORM\Column
-     * @Assert\NotBlank
```

Comment:
Really not needed? I'd have think those are for a given behat test, if not np just remove them though it's not part of the fix (or is it?).

Reply:
Originally, I created this `Foo` entity for my tests. It was copied/pasted from another one. But I think the assertion is useless on this entity as I only create the objects by myself without validation.
This modification (in this PR) is a clean :-)